packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from 'react';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'Stores/connect';
+import ProofOfOwnershipForm from './proof-of-ownership-form.jsx';
+import POOSubmitted from 'Components/poo-submitted';
+import { Loading } from '@deriv/components';
+import POONotRequired from 'Components/poo-not-required';
+import POOVerified from 'Components/poo-verified';
+import POORejetced from 'Components/poo-rejected';
+import { POO_STATUSES } from './constants/constants';
+import paymentMethodConfig from './payment-method-config.js';
+
+export const ProofOfOwnership = ({
+    account_status,
+    client_email,
+    is_dark_mode,
+    refreshNotifications,
+    updateAccountStatus,
+}) => {
+    const cards = account_status?.authentication?.ownership?.requests;
+    const needs_verification = account_status?.authentication?.needs_verification?.includes?.(POO_STATUSES.ownership);
+    const [status, setStatus] = useState(POO_STATUSES.none);
+    const grouped_payment_method_data = React.useMemo(() => {
+        const groups = {};
+        let total_documents_required = 0;
+        cards?.forEach(card => {
+            const card_details = paymentMethodConfig[card.payment_method.toLowerCase()] ?? paymentMethodConfig.other;
+            if (groups[card?.payment_method?.toLowerCase()]) {
+                groups[card?.payment_method?.toLowerCase()].items.push(card);
+                total_documents_required += card?.documents_required;
+            } else {
+                total_documents_required += card?.documents_required;
+                groups[card?.payment_method?.toLowerCase()] = {
+                    icon: is_dark_mode ? card_details?.icon_dark : card_details?.icon_light,
+                    payment_method: card?.payment_method,
+                    items: [card],
+                    instructions: card_details.instructions,
+                    input_label: card_details?.input_label,
+                    identifier_type: card_details.identifier_type,
+                    is_generic_pm: !card_details?.input_label,
+                };
+            }
+        });
+        return { groups, total_documents_required };
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
is it possible to remove this eslint-disable rule and provide all of the necessary things to the dependency array?

Reply:
Yes @ako, it's possible, but that would also mean that we'll have to run the `callback` `function` every time a user switches from `light` mode to `dark` mode and vice versa. Is that ok?

Reply:
the thing is moving forward we have to handle all of these things to migrate to react 18
I guess there would be other ways that we can handle it without any side effect.
we can investigate it together if you think that can help

Reply:
Alright. I'll reach out to you later. Let me leave it for now, until we come to a conclusion

Reply:
@ali-hosseini-fs, I'm going to add `is_dark_mode` to the dependencies, because I'll need to load the appropriate icon when `dark mode` is  turned on, therefore, I'll remove the `// eslint-disable-next-line react-hooks/exhaustive-deps`